src/components/Footer/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Footer: React.FC = () => {
+    return (
+        <div className="bg-gray-200 flex justify-center items-center h-[6vh]">
```

Comment:
h-[6vh] style on the .bg-gray-200 flex justify-center items-center div will cause the footer to be 6% of the viewport height, even if the viewport is less than 60 pixels high. This means that the footer will overlap the content of the website on small screens.

Reply:
https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/75125943/77926179-e3dc-42db-b58a-344e55238e0e